config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml

```diff
@@ -28,13 +44,18 @@ spec:
   - supported: true
     type: AllNamespaces
   keywords:
-  - orchestrator
+  - Orchestrator
+  - RHDH
+  - Backstage
   links:
   - name: Orchestrator Operator
     url: https://github.com/parodos-dev/orchestrator-helm-operator
+  maintainers:
+  - email: jgil@redhat.com
+    name: Jordi Gil
   maturity: alpha
   minKubeVersion: 1.27.0
   provider:
     name: Red Hat
     url: https://parodos.dev
-  version: 0.0.1
+  version: 0.0.5
```

Comment:
Shouldn't it be 0.0.6?

Reply:
Done. I've also pushed a new version of the CSV generated by the bundle. Again not needed as it is updated every time `make bundle` is run, but I wanted to keep things tidy.